modules/olm-subscriptions/chart/templates/functionmesh.yaml

```diff
@@ -23,15 +23,15 @@ metadata:
 spec:
   channel: {{ .Values.functionMesh.channel | default .Values.channel }}
   config:
+    {{- if .Values.tolerations }}
+    tolerations:
+{{ toYaml .Values.tolerations | indent 4 }}
+    {{- end }}
     env:
     - name: ENABLE_WEBHOOKS
       value: "{{ .Values.functionMesh.config.enableWebhooks }}"
     - name: ENABLE_FUNCTION_MESH_CONTROLLER
       value: "{{ .Values.functionMesh.config.enableController }}"
-    {{- if .Values.tolerations }}
-    tolerations:
-{{ toYaml .Values.tolerations | indent 4 }}
-    {{- end }}
```

Comment:
It seems this block actually won't be added to the `env` block.

See below test

```shell
helm template test . --show-only templates/functionmesh.yaml --set "tolerations[0].key=key1" --set "tolerations[0].effect=NoSchedule"
```

rendered out

```yaml
---
# Source: olm-subscriptions/templates/functionmesh.yaml
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: function-mesh
  namespace: sn-system
spec:
  channel: stable
  config:
    env:
    - name: ENABLE_WEBHOOKS
      value: "true"
    - name: ENABLE_FUNCTION_MESH_CONTROLLER
      value: "false"
    tolerations: 
    - effect: NoSchedule
      key: key1
    resources: 
      limits:
        cpu: 200m
        memory: 256Mi
      requests:
        cpu: 20m
        memory: 16Mi
  installPlanApproval: Automatic  
  source: sn-catalog
  sourceNamespace: olm
  name: function-mesh
```

The result turns out the tolerations block is in parallel with env block.